allennlp/models/model.py

```diff
@@ -231,8 +246,9 @@ def load(cls,
         # want the code to look for it, so we remove it from the parameters here.
         _remove_pretrained_embedding_params(model_params)
         model = Model.from_params(vocab, model_params)
-        model_state = torch.load(weights_file, map_location=util.device_mapping(cuda_device))
-        model.load_state_dict(model_state)
+        if not model.is_ensemble:
```

Comment:
Alternatively I could handle this by checking whether `Ensemble` is a base class.